.github/workflows/qiita-plugin-ci.yml

```diff
@@ -79,7 +79,7 @@ jobs:
           # seqtk is a conda-installed requirement for metapool and isn't
           # installed automatically by its setup.py.
           conda config --add channels bioconda
-          conda create -q --yes -n klp python=3.9 seqtk
+          conda create -q --yes -n klp python=3.9 'seqtk>=1.4'
```

Comment:
should we open an issue to make sure we can use more recent versions?

Reply:
It should be fine. We only use seqtk to count sequences and I don't believe that would change from version to version. We're explicitly adding >= version 1.4 here because earlier versions don't have that functionality.

Reply:
Thank you for the clarification, I read that the wrong way (like 'seqtk<=1.4').